src/main/webapp/app/shared/markdown-editor/ace-editor/ace-editor.component.ts

```diff
@@ -5,6 +5,21 @@ import 'brace/theme/monokai';
 import 'brace/theme/chrome';
 import 'brace/theme/dreamweaver';
 import 'brace/theme/dracula';
+import 'brace/ext/language_tools';
+import 'brace/ext/modelist';
+import 'brace/mode/java';
+import 'brace/mode/sh';
+import 'brace/mode/markdown';
+import 'brace/mode/haskell';
+import 'brace/mode/ocaml';
+import 'brace/mode/c_cpp';
+import 'brace/mode/python';
+import 'brace/mode/swift';
+import 'brace/mode/yaml';
+import 'brace/mode/makefile';
+import 'brace/mode/kotlin';
+import 'brace/mode/assembly_x86';
+import 'brace/mode/vhdl';
```

Comment:
Do we have to update these imports whenever we add a new programming language? If so, we should probably note that down somewhere, since this can easily be overlooked

Reply:
The goal of my thesis is to get rid of Ace entirely, so this file won't exist for too much longer. I don't expect that we'll be adding any new modes to the Ace-based editors